doc/source/whatsnew/v1.2.0.rst

```diff
@@ -191,6 +191,7 @@ Other enhancements
 - :meth:`DatetimeIndex.searchsorted`, :meth:`TimedeltaIndex.searchsorted`, :meth:`PeriodIndex.searchsorted`, and :meth:`Series.searchsorted` with datetimelike dtypes will now try to cast string arguments (listlike and scalar) to the matching datetimelike type (:issue:`36346`)
 - Added methods :meth:`IntegerArray.prod`, :meth:`IntegerArray.min`, and :meth:`IntegerArray.max` (:issue:`33790`)
 - Where possible :meth:`RangeIndex.difference` and :meth:`RangeIndex.symmetric_difference` will return :class:`RangeIndex` instead of :class:`Int64Index` (:issue:`36564`)
+- Added :meth:`MultiIndex.dtypes` (:issue:`37062`)
```

Comment:
can you add in the docs in  reference/index.rst as well

Reply:
Do you mean reference/indexing.rst? That's where I see the relevant methods/props documented

Reply:
yes i think it's `reference/indexing.rst`

Reply:
I have added `MultiIndex.dtypes` to `reference/indexing.rst` in [this commit](https://github.com/pandas-dev/pandas/pull/37073/commits/de56f049f05e63f661268db914c3241b31ba87bf#)